custom-login/src/App.test.jsx

```diff
@@ -0,0 +1,28 @@
+/* global jest, beforeEach, afterEach, it, expect */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import App from './App';
+
+jest.mock('./Login', () => () => <div>Login</div>);
+
+let container;
+
+beforeEach(() => {
+  container = document.createElement('div');
```

Comment:
What is this for?

Reply:
it's copied from https://github.com/mraible/schematics/blob/main/src/add-auth/react/src/App.test.js
looks like `mount` should be enough here.

Reply:
after checking the doc, it's the suggested way from `react test utils`
Ref: https://reactjs.org/docs/test-utils.html#act